specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/examples/ApiManagementListApiRevisions.json

```diff
@@ -11,6 +11,7 @@
       "body": {
         "value": [
           {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/a1;rev=1/revisions/a1;rev=1",
```

Comment:
>": [](start = 15, length = 2)

This is wrong and violates ARM Contract.
Should be like this instead.

{
   `"id"`: "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/a1;rev=1/revisions/a1;rev=1",

   `"properties"`: {
            "apiId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/a1;rev=1",
            "apiRevision": "1",
            "createdDateTime": "2018-02-01T22:21:20.467Z",
            "updatedDateTime": "2018-02-01T22:21:20.467Z",
            "isOnline": true,
            "isCurrent": true
     }
}

Reply:
We have a set of "NonArmStyleEntities" described here:
public static readonly HashSet<string> NonArmStyleEntities = new HashSet<string>(new[] {    ...

Those entities don't have "properties" yet. ARM never complained about this, the only issue was "id property is missing"